pkg/bootstrap/docker/up.go

```diff
@@ -511,6 +519,42 @@ func (c *ClientStartConfig) EnsureHostDirectories(io.Writer) error {
 	return c.HostHelper().EnsureVolumeShare()
 }
 
+// EnsureDefaultRedirectURIs merges a default URL to an auth client's RedirectURIs array
+func (c *ClientStartConfig) EnsureDefaultRedirectURIs(out io.Writer) error {
+	oc, _, err := c.Clients()
+	if err != nil {
+		return nil
+	}
+
+	webConsoleOAuth, err := oc.OAuthClients().Get(defaultRedirectClient)
+	if err != nil {
+		if kerrors.IsNotFound(err) {
+			fmt.Fprintf(out, "Unable to find OAuthClient %q\n", defaultRedirectClient)
+			return nil
+		}
+		return err
```

Comment:
@liggitt Sounds good, it no longer returns the `err` but rather prints a slightly updated suggestion and returns `nil`